metrixpp/ext/std/tools/view.py

```diff
@@ -657,3 +660,87 @@ def cout_txt(data, loader):
                 "Directory content:",
                 details)
     
+def cout_prom_regions(path, regions, indent = 0):
+    for region in regions:
+        details = []
+        for namespace in sorted(list(region['data'].keys())):
+            diff_data = {}
+            if '__diff__' in list(region['data'][namespace].keys()):
+                diff_data = region['data'][namespace]['__diff__']
+            for field in sorted(list(region['data'][namespace].keys())):
+                diff_str = ""
+                if field == '__diff__':
+                    continue
+                if field in list(diff_data.keys()):
+                    diff_str = " [" + ("+" if diff_data[field] >= 0 else "") + str(diff_data[field]) + "]"
+                details.append((namespace + ":" + field, str(region['data'][namespace][field]) + diff_str))
+        promout.notify(path = path,
+                        region = region['info']['name'],
+                        metric = "",
+                        details = details)
+        if 'subregions' in list(region.keys()):
+            cout_txt_regions(path, region['subregions'], indent=indent+1)
+            
+def cout_prom(data, loader):

```

Comment:
I trust you know what you are doing here and can be available to help supporting this part of the code related to Prometheus output. I have only briefly looked into the logic of this part and left some comments. If you add more comprehensive tests to cover the case of output with regions / subregions, it would be more robust and the best verification.

I hope @prozessorkern can have a look into this bit too.

Reply:
As said, it seems IMO just to be dead code as the view functionality does not need it to produce what I expect to get in term on prometheus-formatted metrics 1+2+3 as put above. But maybe I missed something or you know a way to get them all in one go?

Reply:
What do you expect the prometheous output to include?

Reply:
See current output on #34 (corresponds to the target output I propose)